crates/ln-dlc-node/src/tests/mod.rs

```diff
@@ -53,7 +53,7 @@ fn init_tracing() {
     TRACING_TEST_SUBSCRIBER.call_once(|| {
         tracing_subscriber::fmt()
             .with_env_filter(
-                "debug,hyper=warn,reqwest=warn,rustls=warn,bdk=info,lightning=debug,sled=info,lightning::chain::channelmonitor=trace",
+                "debug,hyper=warn,reqwest=warn,rustls=warn,bdk=info,lightning=debug,sled=info,ln_dlc_node=debug,ln_dlc_node::ln=debug,lightning::chain::channelmonitor=trace",
```

Comment:
:question: Are you sure this makes a difference? If we specify `debug` at the front and no other pattern references `ln_dlc_node` I don't get how this can activate new logs.

Reply:
I was under impression that these logs were missing.... but maybe I was wrong? I will remove it from this changeset, let me know if you ever miss these logs ;D